doc/profile_changes_SLE15.md

```diff
@@ -0,0 +1,473 @@
+## Main differences between SLE12 and SLE15 profiles
+
+### Product selection
+
+Starting with SLE15, all products are distributed using one medium.
+You need to choose which product to install. To do so explicitly, use the
+'product' option.
+
+#### Explicit Product Selection
+
+```xml
+<software>
+  <products config:type="list">
+    <product>SLED</product>
+  </products>
+</software>
+```
+
+ In special cases, the medium might contain only one product. If so, you
+ do not need to select a product explicitly as described above. AutoYaST will
+ select the only available product automatically.
+
+ For backward compatibility with profiles created for pre-SLE 15 products, AutoYaST
+ implements a heuristic that selects products automatically. This heuristic will be
+ used when the profile does not contain a 'product' element. This heuristic is based
+ on the package and pattern selection in the profile. However, whenever possible,
+ avoid using this mechanism and adapt old profiles to use explicit product selection.
```

Comment:
Let's add something like: "If product selection fails error is shown and installation cannot be performed".

Reply:
:heavy_check_mark: Fixed below

Reply:
Yep, sorry I forgot to mention :+1: